### PR TITLE
Fix build error in artik053/hello-protected

### PR DESCRIPTION
**Pull request by kishore-sn**
Disable CONFIG_ENABLE_HEAPINFO_CMD since HEAPINFO is not enabled in kernel.

Signed-off-by: Kishore SN <kishore.sn@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [25dffb27ab6bb5803a657a7646c6a38aae10695e] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [25dffb27ab6bb5803a657a7646c6a38aae10695e] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@kishore-sn Could you update defconfig with latest Kconfig? It has `# CONFIG_XX=y`. It means it was modified manually. That is what we prohibit.

**Comment by jeongchanKim** [comment]
If HEAPINFO is not enabled in kernel, then `CONFIG_DEBUG_MM_HEAPINFO` should be also disabled.

**Comment by kishore-sn** [comment]
I will raise a new PR with suggested changes.

**Comment by kishore-sn** [comment]
@sunghan-chang 
I updated the defconfig with latest kconfig values.

**Comment by seinfra** [comment]
Target : [04fbbda28882e76e0a4169d79d6fa679745eb532] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [04fbbda28882e76e0a4169d79d6fa679745eb532] - Code Rule Check (C++) OK.

File: os/board/common/Kconfig [hunk 1]
```diff
@@ -31,20 +31,20 @@ config FLASH_MINOR
...
-	default "none,none,none,none,none,none,none,none,smartfs,config,none,"
+	default "none,none,none,none,none,none,none,none,none,smartfs,config,none,"
 	---help---
 		Comma separated list of partition types that can be one of
 		followings: none, smartfs, config
 
 config FLASH_PART_NAME
 	string "Flash partition name list"
-	default "bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,nvram,sssrw,"
+	default "bl1,sssro,bl2,sssfw,wlanfw,os,apps,factory,ota,user,nvram,sssrw,"
```
Comment (sunghan-chang) [inline os/board/common/Kconfig]: protected build is not a default.
Let's use two defaults with conditional.
```
default "xxxxxx" if !BUILD_PROTECTED
default "yyyyyy" if BUILD_PROTECTED
```

Comment (kishore-sn) [reply os/board/common/Kconfig]: ok. Done

**Comment by seinfra** [comment]
Target : [1f98130ae3c3624d978dc7e581a212031428ff71] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [1f98130ae3c3624d978dc7e581a212031428ff71] - Code Rule Check (C++) OK.

**Comment by sunghan-chang** [comment]
@kishore-sn Sorry for late request. But I would like to split this as two commits.
One is updating defconfig of artik053/hello-protected, another is adding default values of flash partition in Kconfig for protected build.

**Comment by kishore-sn** [comment]
@sunghan-chang 
I created 2 commits as per your suggestion.

**Comment by seinfra** [comment]
Target : [6361a2c0fd97336371348a3aca275f1d2d57ce91] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [6361a2c0fd97336371348a3aca275f1d2d57ce91] - Code Rule Check OK.

**Pull request merged**